NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs

```diff
@@ -325,19 +333,74 @@ public async Task HackAndSlash()
             
         }
 
-        private BlockChain<PolymorphicAction<ActionBase>> GetContextFx(PrivateKey playerPrivateKey, RankingState ranking)
+        [Fact]
+        public async Task QuestReward()
+        {
+            var playerPrivateKey = new PrivateKey();
+            Address playerAddress = playerPrivateKey.ToAddress();
+            var ranking = new RankingState();
+            for (var i = 0; i < RankingState.RankingMapCapacity; i++)
+            {
+                ranking.RankingMap[RankingState.Derive(i)] = new HashSet<Address>().ToImmutableHashSet();
+            }
+            var blockChain = GetContextFx(playerPrivateKey, ranking);
+
+            var createAvatarQuery = $"mutation {{ action {{ createAvatar }} }}";
+            await ExecuteQueryAsync(createAvatarQuery);
+            await blockChain.MineBlock(playerAddress);
+
+            var playerState = (Bencodex.Types.Dictionary)blockChain.GetState(playerAddress);
+            var agentState = new AgentState(playerState);
+            var weeklyArenaAddress = (new WeeklyArenaState(1)).address;
+            var rankingMapAddress = RankingState.Derive(0);
+            var gameConfigState = new GameConfigState((Bencodex.Types.Dictionary)blockChain.GetState(Addresses.GameConfig));
+            var questRow = _tableSheets.WorldQuestSheet.First;
```

Comment:
What means this line? If `First` means the specific row, is there another way to find the row though the table sheet changed? If it is, I believe it will be better to use the way.

Reply:
It is intended to get specific row, and its predicate was missing. Fixed it!